resources/views/instructions.blade.php

```diff
@@ -0,0 +1,216 @@
+@extends('layouts.app')
+
+@section('content')
+
+@include('partials.sidebar')
+
+<div class="container">
+	<div class="card">
+		<div class="card-header">
+			<h3 class="card-title">Instructions</h5>
+		</div>
+		<div class="card-body" data-spy="scroll" data-target=".navbar" data-offset="50">
+			<h5 class="card-text" style="color:black;">Using Evidencio's <b>Designer</b>, you can integrate your Evidencio Models to Evidencio Patient Portal. These integrated models are called <b>Workflows</b>.</h5>
+
+			<br>
+
+			<div class="card" style="width: 25rem;">
+				<div class="card-body">
+			<h5><b>Content</b></h5>
+  				<ol>
+   					 <li><a href="#section1"><h5 class="card-text" style="color:blue;">Add a Title and a Description</h5></a></li>
+   					 <li><a href="#section2"><h5 class="card-text" style="color:blue;">Find your Evidencio Model</h5></a></li> 
+   					 <li><a href="#section3"><h5 class="card-text" style="color:blue;">The Variables</h5></a></li>
+   					 <li><a href="#section4"><h5 class="card-text" style="color:blue;">Add Levels and Steps</h5></a></li>
+   					 <li>
+   					 	<a href="#section5"><h5 class="card-text" style="color:blue;">Modify the Step</h5></a>
+   					 	<ol>
+   					 		<li><a href="#section7"><h5 class="card-text" style="color:blue;">Style Your Step</h5></a></li>
+   					 		<li><a href="#section8"><h5 class="card-text" style="color:blue;">Variables Menu</h5></a></li>
+   					 		<li><a href="#section9"><h5 class="card-text" style="color:blue;">Model Calculation Menu</h5></a></li>
+   					 		<li><a href="#section10"><h5 class="card-text" style="color:blue;">Logic Menu</h5></a></li>
+   					 	</ol>	
+   					 </li>
+
+   					 <li><a href="#section6"><h5 class="card-text" style="color:blue;">Save your Workflow</h5></a></li>
+   				</ol>
+		</div>
+	</div>
+
+
+			<br>
+
```

Comment:
Nice style! Some problems though:

The information is nice, the pictures are sadly outdated, for example:
-The model-search is now moved to a modal
-The looks of the steps editing looks very different.
-Result steps can now also be edited, though these might change a bit still


Also, could you perhaps make the images scale with the pagesize? Only if you're done with everything else though